src/flags.cpp

```diff
@@ -0,0 +1,38 @@
+#include "../include/flags.h"
```

Comment:
You shouldn't use relative paths in includes. Instead use the `-I` compiler argument.